packages/marketplace/contracts/TransferManager.sol

```diff
@@ -33,6 +32,8 @@ abstract contract TransferManager is Initializable, ITransferManager {
     /// @notice cannot exceed 50% == 0.5 * BASE_POINTS == 5000
     uint256 internal constant ROYALTY_SHARE_LIMIT = 5000;
 
+    uint256 internal constant GRID_SIZE = 408;
```

Comment:
We're fully certain this will never change?

Reply:
Refer to : https://github.com/thesandboxgame/sandbox-smart-contracts/blob/master/packages/land/contracts/common/LandBaseToken.sol

Reply:
And if they change it and upgrade the contract we will be screwed, can we get that information from the land contract by calling:

```solidity
    /// @notice total width of the map
    /// @return width
    function width() external pure virtual returns (uint256) {
        return GRID_SIZE;
    }

    /// @notice total height of the map
    /// @return height
    function height() public pure returns (uint256) {
        return GRID_SIZE;
    }
 ```